modules/tensor_mechanics/src/actions/PressureAction.C

```diff
@@ -42,11 +42,17 @@ validParams<PressureAction>()
   params.addParam<Real>("factor", 1.0, "The factor to use in computing the pressure");
   params.addParam<Real>("alpha", 0.0, "alpha parameter for HHT time integration");
   params.addParam<FunctionName>("function", "The function that describes the pressure");
+  params.addParam<AuxVariableName>("pressure", "The auxiliary variable that contains the pressure");
```

Comment:
I'm not crazy about this `pressure` parameter name because it doesn't jump out at me as being the option that you would use to specify pressure via a coupled variable.

Reply:
Then, pick a better name. I do not really care. I was going for the syntax that would look like variable coupling...

Reply:
How about `coupled_pressure`? Also, the description should be changed to drop 'auxiliary' because it could be a solution variable.